templates/oam/workloads/stable-2.11.0/serverauthorization.linkerd.meshery.layer5io.schema.json

```diff
@@ -181,8 +181,5 @@
       }
     }
   },
-  "$schema": "http://json-schema.org/draft-04/schema#",
-  "version": "stable-2.11.0",
-  "object-type": "ServerAuthorization",
```

Comment:
As this is removed here, where will we now be saving this metadata instead?

Reply:
This is not part of the spec. That is why we are removing these three fields slowly

Reply:
Ok, so we source these names from the definition.json instead, right?